polkadot-protocol/architecture/polkadot-chain/overview.md

```diff
@@ -1,4 +1,371 @@
+
+Crowdloans enabled users to securely lend funds to teams for lease deposits in exchange for pre-sale tokens, which is the only way to access slots on Polkadot 1.0.
+
+!!!note
+    Auction are deprecated in favor of Coretime
+
+Additionally, the  chain handles [staking](add link to PoS page){target=\_blank}, [accounts](add a link to accounts pages){target=\_blank}, balances, and [governance](add a link to govenance page){target=\_blank}.
+
+### Polkadot's Resilience
+
+Decentralization is a vital component of blockchain networks, but it comes with trade-offs:
+
+- An overly decentralized network may face challenges in reaching consensus and require significant energy to operate.
+- Also a network that achieves consensus quickly risks centralization, making it easier to manipulate or attack.
+
+Ideally, a network should be decentralized enough to prevent any manipulative or malicious influence. In this sense, decentralization serves as a tool to achieve resilience. 
+
+**Polkadot 1.0** currently achieves resilience through several strategies:
+
+1. **Nominated Proof of Staking (NPoS)**: This ensures that the stake per validator is maximized and evenly distributed among validators.
+
+2. **1KV Program**: Designed to encourage new operators to join the network, this program enhances both physical decentralization (the number of validator nodes per service provider) and social decentralization (the number of validator nodes per operator). Feel free to explore these metrics using the [Polkawatch App](https://polkadot.polkawatch.app/){target=\_blank}.
+
+3. **On-Chain Treasury and Governance**: Known as [OpenGov](link to opengov overview page){target=\_blank}, this system allows every decision to be made through public referenda, enabling any token holder to cast a vote.
+
+### Polkadot's Blockspace
+
+Polkadot 1.0’s design allows for the commoditization of blockspace.
+
+Blockspace is the capacity of a blockchain to finalize and commit operations, encompassing its security, computing, and storage capabilities. Its characteristics can vary across different blockchains, affecting **security**, **flexibility**, and **availability**.
+
+- **Security** measures the robustness of blockspace in Proof-of-Stake (PoS) networks, linked to the stake locked on validator nodes, the variance in stake among validators, and the total number of validators. It also considers social centralization (how many validators are owned by single operators) and physical centralization (how many validators run on the same service provider).
+
+- **Flexibility** reflects the functionalities and types of data that can be stored, with high-quality data essential to avoid bottlenecks in critical processes.
+
+- **Availability** indicates how easily users can access blockspace. It should be easily accessible, allowing diverse business models to thrive, ideally regulated by a marketplace based on demand and supplemented by options for "second-hand" blockspace.
```

Comment:
```suggestion
- **Security** - measures the robustness of blockspace in Proof-of-Stake (PoS) networks, linked to the stake locked on validator nodes, the variance in stake among validators, and the total number of validators. It also considers social centralization (how many validators are owned by single operators) and physical centralization (how many validators run on the same service provider)

- **Flexibility** - reflects the functionalities and types of data that can be stored, with high-quality data essential to avoid bottlenecks in critical processes

- **Availability** - indicates how easily users can access blockspace. It should be easily accessible, allowing diverse business models to thrive, ideally regulated by a marketplace based on demand and supplemented by options for "second-hand" blockspace
```